python-client/giskard/core/model_validation.py

```diff
@@ -191,12 +191,16 @@ def validate_label_with_target(classification_labels, target_values=None, target
                 'to make results more understandable in Giskard."'
             )
 
+        _to_append = ""
```

Comment:
Not an urgent  improvement, but I think if we're going to add arbitrary_types_allowed every time, it's just worth wrapping validate_arguments with our own decorator that sets `{'arbitrary_types_allowed': True}` by default

Reply:
done here: https://github.com/Giskard-AI/giskard/blob/fddd931f062065e47db3876291c18a6fe4251963/python-client/giskard/core/validation.py#L6-L20